go.mod

```
@@ -2,6 +2,8 @@ module github.com/ava-labs/teleporter
 
 go 1.20
 
+// GO_PATCH_VERSION 10
```

Why not this, so that `go mod tidy` can enforce it?
```suggestion
go 1.20.10
```

---

So this is a little weird... tests work locally for me when this is set, [but on CI, it fails](https://github.com/ava-labs/teleporter/actions/runs/6906754471/job/18792474985). Everything I can find has the `go` version in `go.mod` without a patch version.

It looks like we could use the [toolchain command,](https://go.dev/doc/toolchain) but it seems like too much work for now.

---

I haven't tried it myself, but this https://github.com/nektos/act allows you to run github actions locally to help debug

---

Unfortunately it seems like `act` is incompatible with `$GITHUB_ENV`, which means I can't run this locally.